src/Spout/Writer/ODS/Manager/WorksheetManager.php

```diff
@@ -125,7 +126,13 @@ public function addRow(Worksheet $worksheet, Row $row)
             $nextCell = isset($cells[$nextCellIndex]) ? $cells[$nextCellIndex] : null;
 
             if ($nextCell === null || $cell->getValue() !== $nextCell->getValue()) {
-                $data .= $this->applyStyleAndGetCellXML($cell, $rowStyle, $currentCellIndex, $nextCellIndex);
+                $registeredStyle = $this->applyStyleAndRegister($cell, $rowStyle);
+                $cellStyle = $registeredStyle->getStyle();
+                if ($registeredStyle->isRowStyle()) {
```

Comment:
I don't understand this `if`. `$registerStyle->isRowStyle()` returns true when cell style is empty (and we therefore use the row style). So isn't it already what the `$rowStyle = $row->getStyle();` returns?

Maybe worth adding a comment here to explain the logic